### ISSUE TITLE
Proofread AI generated documentation

### ISSUE THREAD

**Issue by zdilarr** [ilia-brykin/aloha#115]
We have recently tried out JetBrains AI assistant in order generate documentation and tests on filters and util functions. 
This generated documentation must now be proofread and, if necessary, revised.

Finally an evaluation can be made if this tool is suitable for future use in aloha.

**Comment by zdilarr** [comment]
Some thoughts:

- Generally does a good job
- Grammar is perfect. Sentences short and to the point and not too detailed or exhaustive.
-  It does miss some things. Example: 
`Filterfilesize cannot be used to convert value from one to another unit of measurement. Not just any meaasurement is supported - Filesize pertains specifically to units of digital information storage. It would not work for meters. Documentation failed to make that clear.`
- I find it very important to note that I found nothing outright false, it does not make up things that do not exist. 
- my personal impression is that it does not seem to utilise variable names too much for additional context, but also does not get confused by weird naming. our names are pretty good so I am not certaint
-  It provides different definitions for same parameter across different filters (options). Not consistent in that area. Examples:
`The translation options. Additional options for the function. Optional configuration. The options object. etc `
- If we were to continue using it, someone must go over generated docs manually. But does the bulk of the work itself.
- It does get scarily close to our manually written documentation sometimes (For example it did a better job than all of us for filterPropertyByValue.js). Maybe we can utilize at lease some of the definitions for our docs Page. English would be a starting point then instead of german, and other languages could follow.

Im voting yes for it personally. 

I touched docs up in few places.

@ilia-brykin is this feedback useful? Ill assign it to you to decide if we were to use it further. Feel free to close ticket youself.
